src/main/java/basic/Exercise08.java

```diff
@@ -0,0 +1,36 @@
+package basic;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Exercise08 {
+  private static final Map<Integer, String> map = new LinkedHashMap<>();
+
+  public Exercise08() {
+    map.put(1000, "M");
+    map.put(900, "CM");
+    map.put(500, "D");
+    map.put(400, "CD");
+    map.put(100, "C");
+    map.put(90, "XC");
+    map.put(50, "L");
+    map.put(40, "XL");
+    map.put(10, "X");
+    map.put(9, "IX");
+    map.put(5, "V");
+    map.put(4, "IV");
+    map.put(1, "I");
+  }
+
+  public static String intToRoman(final int n) {
+    StringBuilder roman = new StringBuilder();
+    int num = n;
+    for (Integer key : map.keySet()) {
+      while (num >= key) {
+        roman.append(map.get(key));
+        num -= key;
+      }
+    }
+    return roman.toString();
```

Comment:
Thêm mấy dòng trống này nữa là ok

```suggestion

    for (Integer key : map.keySet()) {
      while (num >= key) {
        roman.append(map.get(key));
        num -= key;
      }
    }
    
    return roman.toString();
```